Controller/BasePlayerController.php

```diff
@@ -21,7 +21,9 @@ public function magicAction(MultimediaObject $multimediaObject, Request $request
             return $this->redirect($this->generateUrl('pumukit_videoplayer_magicindex', array('id' => $multimediaObject->getSecret(), 'secret' => $multimediaObject->getSecret())).'&secret='.$multimediaObject->getSecret());
         }
 
-        $response = $this->testBroadcast($multimediaObject, $request);
+        $embeddedBroadcastService = $this->get('pumukitschema.embeddedbroadcast');
+        $password = $request->get('broadcast_password');
+        $response = $embeddedBroadcastService->canUserPlayMultimediaObject($multimediaObject, $this->getUser(), $password, $request->query->get('force-auth'));
```

Comment:
`force-auth` is not needed.

See https://github.com/campusdomar/PuMuKIT2/commit/8d074b045ee28f3e609e645e027b93a949180291